docs/source/upgrade-guide/index.md

```diff
@@ -1,17 +1,59 @@
 # Upgrade Guide
 
 This upgrade guide lists all breaking changes in Volto and explains the
- steps that are necessary to upgrade to the lastest version.
+steps that are necessary to upgrade to the lastest version.
 
 !!! note
-    There are times that updating the Volto boilerplate (the one generated by
```

Comment:
Don't strip that leading whitespace, it's important. See https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage

Reply:
ah, my eslint does that. I'm seeing it in rest of my code. I have to disable eslint for this file.

Reply:
@tiberiuichim Done!